boards/limifrog-v1/include/lsm6ds3_params.h

```diff
@@ -0,0 +1,48 @@
+/*
```

Comment:
Can this file be moved to `drivers/lsm6ds3/include` so one can use it with the default params and the sensor plugged on another board ?